bitnami/rabbitmq-cluster-operator/Chart.yaml

```
@@ -26,4 +26,4 @@ name: rabbitmq-cluster-operator
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/rabbitmq-cluster-operator
   - https://github.com/rabbitmq/cluster-operator
-version: 3.1.5
+version: 3.1.6
```

Thanks for the PR. Since this PR adds a new feature instead of fixing a bug, could you please the chart version in a minor instead of a patch? Thx 

```suggestion
version: 3.2.0
```

---

@carrodher it's done :slightly_smiling_face: